### PR TITLE
feat: fix pricing issue

### PR DESCRIPTION
**Pull request by wannacfuture**

### CODE REVIEW

File: src/handlers/pricing/pricing-label.ts [hunk 1]
```diff
@@ -74,10 +74,12 @@ export async function onLabelChangeSetPricing(context: Context): Promise<void> {
     return;
   }
 
+  if (!hasAssistivePricing) return;
```
Comment (0x4007) [inline src/handlers/pricing/pricing-label.ts:77]: I think that `void` returns are banned in favor of `null`. Are you sure this runs without issue?

Comment (wannacfuture) [reply src/handlers/pricing/pricing-label.ts:77]: Yeah, when I tried, there was no issue.

**Comment by 0x4007** [comment]
Make sure to fix CI. I think this was my fault. I spent an afternoon reading yarn docs and made some changes that seem out of sync with your fork.

**Comment by wannacfuture** [comment]
```
Make sure to fix CI. I think this was my fault. I spent an afternoon reading yarn docs and made some changes that seem out of sync with your fork.
```
What do you mean?
Do you what me to fix these things? 
https://github.com/pavlovcik/ubiquibot/actions/runs/7460859175/job/20299792303?pr=63
https://github.com/pavlovcik/ubiquibot/actions/runs/7460775894/job/20299541209?pr=63
@pavlovcik

**Comment by 0x4007** [comment]
> ```
> Make sure to fix CI. I think this was my fault. I spent an afternoon reading yarn docs and made some changes that seem out of sync with your fork.
> ```
> 
> What do you mean? Do you what me to fix these things? https://github.com/pavlovcik/ubiquibot/actions/runs/7460859175/job/20299792303?pr=63 https://github.com/pavlovcik/ubiquibot/actions/runs/7460775894/job/20299541209?pr=63 @pavlovcik

I vaguely remember what I did a few weeks ago but a good starting point to fix this is to check out https://yarnpkg.com/corepack

**Comment by 0x4007** [comment]
You can fix CI in a new PR

**Pull request merged**